### ISSUE TITLE
Create blog with hugo.

### ISSUE THREAD

**Issue by zo0o0ot** [Leagify/blog#1]

**Comment by yamilurbina** [comment]
@zo0o0ot I'm very interested in helping with this, but I have some questions:

* Any blog or blogs you'd like to show as example? 
* Any Hugo theme you'd like to use?

**Comment by zo0o0ot** [comment]
I'm thinking ananke.  I've taken a look at some other themes, and that seems reasonably good for what I'm hoping to do.  The main thing I need is a reasonably easy to configurable main page and the ability to easily create a sub page, which ananke seemed to do reasonably well.

Background:
I have two sites set up currently:
http://leagify.me/
http://draftcharts.azurewebsites.net/

I'm looking to move both of those sites to a single github pages site, created with Hugo.  I've recently created a different site at https://github.com/sermonarchives/blog located at https://sermonarchives.github.io/ and I'm hoping to leverage some of what I've learned to help with this site, too.


As far as I can tell, I have an open PR for a skeleton of this, but I'll probably need to make some modifications to make things better, so I'll make sure to keep tagging with hacktoberfest, and I'll make sure to at you, if that's OK.
